### PR TITLE
7za, 7zr, [: update pt_BR translation

### PR DESCRIPTION
**Pull request by barbaraips**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/7za.md [hunk 1]
```diff
@@ -8,18 +8,30 @@
 
 `7za a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages.pt_BR/common/7za.md:9]: In this example this is a better translation:
```suggestion
`7za a {{caminho/para/arquivo_compactado.7z}} {{caminho/para/arquivo_ou_diretório}}`
```

File: pages.pt_BR/common/7za.md [hunk 2]
```diff
@@ -8,18 +8,30 @@
...
+
+- Compactar utilizando um tipo específico de arquivamento/compressão:
+
+`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Exibir o conteúdo de um arquivo:
 
 `7za l {{arquivo.7z}}`
+
+- Exibir os tipos de arquivamento/compressão disponíveis:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:35]: For Romance languages, the present indicative fits better.
```suggestion
- Exibe os tipos de arquivamento/compressão disponíveis:
```
Please apply this to the other examples.

**Comment by barbaraips** [comment]
Thanks @kbdharun and @navarroaxel , commit updated

File: pages.pt_BR/common/7za.md [hunk 3]
```diff
@@ -6,20 +6,32 @@
 
 - Compactar um arquivo ou diretório:
 
-`7za a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za a {{caminho/para/arquivo_compactado.7z}} {{caminho/para/arquivo_ou_diretório}}`
+
+- Criptografar um arquivo existente (incluindo cabeçalhos):
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:11]:
```suggestion
- Criptografa um arquivo existente (incluindo cabeçalhos):
```

File: pages.pt_BR/common/7za.md [hunk 4]
```diff
@@ -6,20 +6,32 @@
...
+
+`7za a {{caminho/para/arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
 
 - Descompactar um arquivo mantendo a estrutura de diretórios original:
 
 `7za x {{arquivo_compactado.7z}}`
 
-- Compactar utilizando um tipo específico de arquivamento/compressão:
+- Descompactar um arquivo em um diretório específicado pelo usuário:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:19]:
```suggestion
- Descompacta um arquivo em um diretório específicado pelo usuário:
```

File: pages.pt_BR/common/7za.md [hunk 5]
```diff
@@ -6,20 +6,32 @@
...
 
-`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za x {{caminho/para/arquivo.7z}} -o{{caminho/para/diretório}}`
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Descompactar um arquivo para a saída padrão:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:23]:
```suggestion
- Descompacta um arquivo para a saída padrão:
```

File: pages.pt_BR/common/7za.md [hunk 6]
```diff
@@ -6,20 +6,32 @@
...
 
-`7za i`
+`7za x {{path/to/archive.7z}} -so`
+
+- Compactar utilizando um tipo específico de arquivamento/compressão:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:27]:
```suggestion
- Compacta utilizando um tipo específico de arquivamento/compressão:
```

File: pages.pt_BR/common/7za.md [hunk 7]
```diff
@@ -6,20 +6,32 @@
...
+
+`7za a -t{{zip|gzip|bzip2|tar}} {{caminho/para/arquivo.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Exibir o conteúdo de um arquivo:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:31]:
```suggestion
- Exibe o conteúdo de um arquivo:
```

File: pages.pt_BR/common/7zr.md [hunk 8]
```diff
@@ -8,10 +8,26 @@
 
 `7zr a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
 
+- Criptografar um arquivo existente (incluindo cabeçalhos):
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:11]:
```suggestion
- Criptografa um arquivo existente (incluindo cabeçalhos):
```

File: pages.pt_BR/common/7zr.md [hunk 9]
```diff
@@ -8,10 +8,26 @@
...
+
+`7zr a {{arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
+
 - Descompactar um arquivo mantendo a estrutura de diretórios original:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:15]:
```suggestion
- Descompacta um arquivo mantendo a estrutura de diretórios original:
```

File: pages.pt_BR/common/7zr.md [hunk 10]
```diff
@@ -8,10 +8,26 @@
...
 
 `7zr x {{arquivo_compactado.7z}}`
 
+- Descompactar um arquivo em um diretório específicado pelo usuário:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:19]:
```suggestion
- Descompacta um arquivo em um diretório específicado pelo usuário:
```

File: pages.pt_BR/common/7zr.md [hunk 11]
```diff
@@ -8,10 +8,26 @@
...
+
+`7zr x {{arquivo.7z}} -o{{caminho/diretório}}`
+
+- Descompactar um arquivo para a saída padrão:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:23]:
```suggestion
- Descompacta um arquivo para a saída padrão:
```

File: pages.pt_BR/common/7zr.md [hunk 12]
```diff
@@ -8,10 +8,26 @@
...
+
+`7zr x {{path/to/archive.7z}} -so`
+
 - Exibir o conteúdo de um arquivo:
 
-`7zr l {{arquivo_compactado.7z}}`
+`7zr l {{arquivo.7z}}`
+
+- Exibir os tipos de arquivamento/compressão disponíveis:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:31]:
```suggestion
- Exibe os tipos de arquivamento/compressão disponíveis:
```

File: pages.pt_BR/common/7zr.md [hunk 13]
```diff
@@ -8,10 +8,26 @@
...
 
+- Descompactar um arquivo em um diretório específicado pelo usuário:
+
+`7zr x {{arquivo.7z}} -o{{caminho/diretório}}`
+
+- Descompactar um arquivo para a saída padrão:
+
+`7zr x {{path/to/archive.7z}} -so`
+
 - Exibir o conteúdo de um arquivo:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:27]:
```suggestion
- Exibe o conteúdo de um arquivo:
```

File: pages.pt_BR/common/[.md [hunk 14]
```diff
@@ -6,20 +6,32 @@
...
 - Testa se um arquivo existe:
 
 `[ -f "{{caminho/para/arquivo}}" ]`
 
-- Testa se um diretório não existe:
+- Testa se um diretório existe:
 
 `[ ! -d "{{caminho/para/diretorio}}" ]`
 
+- Testa se um determinado arquivo ou diretório existe:
```
Comment (navarroaxel) [inline pages.pt_BR/common/[.md:31]:
```suggestion
- Testa se um determinado arquivo ou diretório [e]xiste:
```

File: pages.pt_BR/common/[.md [hunk 15]
```diff
@@ -6,20 +6,32 @@
...
 
-`[ -z "{{$GIT_BRANCH}}" ]`
+`[ -z "{{variavel}}" ]`
 
 - Testa se um arquivo existe:
 
 `[ -f "{{caminho/para/arquivo}}" ]`
 
-- Testa se um diretório não existe:
+- Testa se um diretório existe:
```
Comment (navarroaxel) [inline pages.pt_BR/common/[.md:27]:
```suggestion
- Testa se um [d]iretório existe:
```

File: pages.pt_BR/common/[.md [hunk 16]
```diff
@@ -6,20 +6,32 @@
...
 - Testa se uma determinada variável é igual a uma determinada string:
 
-`[ "{{$VARIAVEL}}" == "{{/bin/zsh}}" ]`
+`[ "{{variavel}}" == "{{/bin/zsh}}" ]`
+
+- Testa se uma determinada variável é igual/diferente/maior que/menor que/maior ou igual/menor ou igual a um determinado inteiro:
+
+`[ "${{variable}}" -{{eq|ne|gt|lt|ge|le}} {{integer}} ]`
+
+- Testa se uma determinada variável tem um valor não vazio:
```
Comment (navarroaxel) [inline pages.pt_BR/common/[.md:15]:
```suggestion
- Testa se uma determinada variável tem um valor [n]ão vazio:
```

File: pages.pt_BR/common/[.md [hunk 17]
```diff
@@ -6,20 +6,32 @@
...
 
 `[ ! -d "{{caminho/para/diretorio}}" ]`
 
+- Testa se um determinado arquivo ou diretório existe:
+
+`[ -e {{caminho/para/arquivo_ou_diretorio}} ]`
+
 - Declaração de if-else:
 
 `[ {{condicao}} ] && {{echo "verdadeiro"}} || {{echo "falso"}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/[.md:37]: This example was deleted from the English page
```suggestion
```

File: pages.pt_BR/common/7zr.md [hunk 18]
```diff
@@ -8,10 +8,26 @@
...
+
+`7zr x {{arquivo.7z}} -o{{caminho/diretório}}`
+
+- Descompacta um arquivo para a saída padrão:
+
+`7zr x {{path/to/archive.7z}} -so`
+
+- Exibe o conteúdo de um arquivo:
+
+`7zr l {{arquivo.7z}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:29]:
```suggestion
`7zr l {{caminho/para/arquivo_compactado.7z}}`
```

File: pages.pt_BR/common/7zr.md [hunk 19]
```diff
@@ -8,10 +8,26 @@
...
 `7zr x {{arquivo_compactado.7z}}`
 
-- Exibir o conteúdo de um arquivo:
+- Descompacta um arquivo em um diretório específicado pelo usuário:
+
+`7zr x {{arquivo.7z}} -o{{caminho/diretório}}`
+
+- Descompacta um arquivo para a saída padrão:
+
+`7zr x {{path/to/archive.7z}} -so`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:25]:
```suggestion
`7zr x {{caminho/para/arquivo_compactado.7z}} -so`
```

File: pages.pt_BR/common/7zr.md [hunk 20]
```diff
@@ -8,10 +8,26 @@
...
+`7zr a {{arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
+
+- Descompacta um arquivo mantendo a estrutura de diretórios original:
 
 `7zr x {{arquivo_compactado.7z}}`
 
-- Exibir o conteúdo de um arquivo:
+- Descompacta um arquivo em um diretório específicado pelo usuário:
+
+`7zr x {{arquivo.7z}} -o{{caminho/diretório}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:21]:
```suggestion
`7zr x {{caminho/para/arquivo_compactado.7z}} -o{{caminho/para/diretório}}`
```

File: pages.pt_BR/common/7zr.md [hunk 21]
```diff
@@ -8,10 +8,26 @@
 
 `7zr a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:9]:
```suggestion
`7zr a {{caminho/para/arquivo_compactado.7z}} {{caminho/para/arquivo_ou_diretorio}}`
```

File: pages.pt_BR/common/7zr.md [hunk 22]
```diff
@@ -8,10 +8,26 @@
...
 
-- Descompactar um arquivo mantendo a estrutura de diretórios original:
+- Criptografa um arquivo existente (incluindo cabeçalhos):
+
+`7zr a {{arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7zr.md:13]:
```suggestion
`7zr a {{arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{caminho/para/arquivo_compactado.7z}}`
```

File: pages.pt_BR/common/7za.md [hunk 23]
```diff
@@ -6,20 +6,32 @@
...
+
+- Compacta utilizando um tipo específico de arquivamento/compressão:
 
-- Exibir o conteúdo de um arquivo:
+`7za a -t{{zip|gzip|bzip2|tar}} {{caminho/para/arquivo.7z}} {{caminho/arquivo_ou_diretório}}`
 
-`7za l {{arquivo.7z}}`
+- Exibe o conteúdo de um arquivo:
+
+`7za l {{caminho/para/arquivo.7z}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:33]:
```suggestion
`7za l {{caminho/para/arquivo_compactado.7z}}`
```

File: pages.pt_BR/common/7za.md [hunk 24]
```diff
@@ -6,20 +6,32 @@
...
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Descompacta um arquivo para a saída padrão:
 
-`7za i`
+`7za x {{path/to/archive.7z}} -so`
+
+- Compacta utilizando um tipo específico de arquivamento/compressão:
 
-- Exibir o conteúdo de um arquivo:
+`7za a -t{{zip|gzip|bzip2|tar}} {{caminho/para/arquivo.7z}} {{caminho/arquivo_ou_diretório}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:29]:
```suggestion
`7za a -t{{zip|gzip|bzip2|tar}} {{caminho/para/arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
```

File: pages.pt_BR/common/7za.md [hunk 25]
```diff
@@ -6,20 +6,32 @@
...
+- Descompacta um arquivo em um diretório específicado pelo usuário:
 
-`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za x {{caminho/para/arquivo.7z}} -o{{caminho/para/diretório}}`
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Descompacta um arquivo para a saída padrão:
 
-`7za i`
+`7za x {{path/to/archive.7z}} -so`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:25]:
```suggestion
`7za x {{caminho/para/arquivo_compactado.7z}} -so`
```

File: pages.pt_BR/common/7za.md [hunk 26]
```diff
@@ -6,20 +6,32 @@
...
 
 - Descompactar um arquivo mantendo a estrutura de diretórios original:
 
 `7za x {{arquivo_compactado.7z}}`
 
-- Compactar utilizando um tipo específico de arquivamento/compressão:
+- Descompacta um arquivo em um diretório específicado pelo usuário:
 
-`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za x {{caminho/para/arquivo.7z}} -o{{caminho/para/diretório}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:21]:
```suggestion
`7za x {{caminho/para/arquivo_compactado.7z}} -o{{caminho/para/diretório}}`
```

File: pages.pt_BR/common/7za.md [hunk 27]
```diff
@@ -6,20 +6,32 @@
 
 - Compactar um arquivo ou diretório:
 
-`7za a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za a {{caminho/para/arquivo_compactado.7z}} {{caminho/para/arquivo_ou_diretório}}`
+
+- Criptografa um arquivo existente (incluindo cabeçalhos):
+
+`7za a {{caminho/para/arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:13]:
```suggestion
`7za a {{caminho/para/arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{caminho/para/arquivo_compactado.7z}}`
```

File: pages.pt_BR/common/7za.md [hunk 28]
```diff
@@ -6,20 +6,32 @@
...
 
 - Descompactar um arquivo mantendo a estrutura de diretórios original:
 
 `7za x {{arquivo_compactado.7z}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7za.md:17]:
```suggestion
`7za x {{caminho/para/arquivo_compactado.7z}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**